timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -181,7 +181,8 @@ def __init__(
             )
             self.context_length = self.maximum_context_length
 
-        model_path_safe = str.replace(model_path_input, "/", "__")
+        # we truncate the name to avoid long path errors on Windows
+        model_path_safe = str(model_path_input).replace("/", "__").replace(os.path.sep, "__")[-50:]
```

Comment:
Why not simply?

```suggestion
        model_path_safe = model_path_input.replace(os.path.sep, "__")[-50:]
```

Reply:
because hf_hub_ids will still contain forward slashes on windows.

Reply:
got it, makes sense!